pages/events.md

```diff
@@ -4,4 +4,16 @@ slug: events
 type: text
 ---
 
-Ansible events! Things to find us at!
+Ansible events! Things to find us at! Check out the [Ansible Meetup page](https://www.meetup.com/topics/ansible/all/) for our existing meetups, or the [Meetup Organizers Toolkit](https://github.com/ansible-community/meetup/blob/main/meetup_toolkit.md) for starting your own.
+
+## [Ansible Community Day @ Red Hat Summit](https://hackmd.io/0uT8IXjJTH2ZBB28jAQ1fA)
```

Comment:
Please use this hackmd url to default it to view (instead of edit/both) for everyone https://hackmd.io/@ansible-community/ACD2023-Boston